packages/ocaml-protoc/ocaml-protoc.2.0.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "Protobuf compiler for OCaml"
+description: "Protobuf compiler for OCaml"
+maintainer: "Maxime Ransan <maxime.ransan@gmail.com>"
+authors:[
+  "Maxime Ransan <maxime.ransan@gmail.com>"
+]
+homepage: "https://github.com/mransan/ocaml-protoc"
+bug-reports:"https://github.com/mransan/ocaml-protoc/issues"
+dev-repo:"git+https://github.com/mransan/ocaml-protoc.git"
+license: "MIT"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>="4.02.1"}
+  "dune"  {build}
```

Comment:
```suggestion
  "dune"  {>= "1.11"}
```
Dune should not be tagged as a build dependency (see #14266). Moreover your `dune-project` file says this package requires at least dune 1.11

Reply:
Thanks for the suggestion and info. I'll accept it.